sdk/eventhub/ci.yml

```diff
@@ -20,24 +25,28 @@ pr:
   paths:
     include:
     - sdk/eventhub/
+    exclude:
+    - sdk/eventhub/Azure.ResourceManager.EventHubs
+    - sdk/eventhub/Microsoft.Azure.EventHubs
+    - sdk/eventhub/Microsoft.Azure.EventHubs.Processor
+    - sdk/eventhub/Microsoft.Azure.EventHubs.ServiceFabricProcessor
+    - sdk/eventhub/Microsoft.Azure.Management.EventHub
 
 extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: eventhub
+    ExcludeProjects:
+    - Azure.ResourceManager.EventHubs
```

Comment:
The wildcard should be sufficient, but there's one outstanding observation from the initial round of testing where it's not 100% confirmed for one of our script calls.

My approach here was to explicitly define everything that exists today and leave the wild card to provide some level of filtering for anything new. I'm still working with members of the engineering team to investigate and will open a new PR to trim to just the wild card, if possible.

Reply:
Should we add the same wildcard to the exclude lists above?

Reply:
The ones above are owned by the DevOps config and have to be in the raw path format.  Unfortunately, they don't support wildcard patterns.